akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -257,8 +277,12 @@ final class ORSet[A] private[akka] (
   /**
    * INTERNAL API
    */
-  private[akka] def remove(node: UniqueAddress, element: A): ORSet[A] =
-    assignAncestor(copy(elementsMap = elementsMap - element))
+  private[akka] def remove(node: UniqueAddress, element: A): ORSet[A] = {
+    // FIXME handle existing delta
```

Comment:
That is harder but still possible if we do a separate merge algorithm for the negative deltas that is a mirror of the one for adds (mirror with respect to '>' relation for dots).